CI/flatpak/format-manifest.py

```diff
@@ -0,0 +1,32 @@
+import os
+import json
+try:
+	import jq
+except ImportError:
+	print('The python module jq is required to enable the use of jq')
+	exit(1)
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
+if not os.path.isfile(dir_path + '/com.obsproject.Studio.json'):
+	print('The script is not ran in the same folder as the manifest')
+	exit(1)
+
+for root, dirs, files in os.walk(dir_path):
+	for file in files:
+		if not file.endswith('.json'):
+			continue
+
+		print(f"Formatting {file}")
+		# Load JSON file
+		f = open(f'{root}/{file}', 'r')
+		j = json.load(f)
+
+		if file == 'com.obsproject.Studio.json':
+			# Sort module files order in the manifest
+			j = jq.compile(".modules[0:-1] = (.modules[0:-1] | sort)").input(j).first()
```

Comment:
With `jq` I can clearly exclude the final module `"obs"` from the sorting easily.

In your idea if `"obs"` was `"a-obs"` it would be moved a the first element.

Reply:
I tried to avoid using `jq`.